prometheus.yml

```diff
@@ -0,0 +1,13 @@
+global:
```

Comment:
This file is not required for the deployment, prometheus is already configured

Reply:
Removed